### PR TITLE
Wappalyzer plugin

### PR DESCRIPTION
**Pull request by cubitouch**
https://github.com/MicrosoftDX/Vorlonjs/issues/122

### CODE REVIEW

**Comment by deltakosh** [comment]
You are unstoppable :D

**Comment by deltakosh** [comment]
Can you summarize how this plugin works?

**Comment by cubitouch** [comment]
Sure :)

It use [Wappalyzer](https://github.com/AliasIO/Wappalyzer/) to detect technologies used in a website implementation. It is based on script tags (to guess js libs for example), meta tags (to guess CMS), HTTP headers (to guess server type), etc. I used script tags so far, meta tags wont be difficult and I think I have solution for HTTP headers.

Example for http://stackoverflow.com/:
image

**Comment by cubitouch** [comment]
By the way, I had to include a [JSON file](https://github.com/AliasIO/Wappalyzer/blob/master/src/apps.json) into the client side TypeScript file.

How can I load an external JSON file (located on the Server), so I can send it to client to facilitate future updates of Wappalyzer?

**Comment by deltakosh** [comment]
My question is: should it be a plugin actually? Why a web developer would like to know which techno is used in is OWN web site ? :)

**Comment by cubitouch** [comment]
Good point :)

It could be useful for a dashboard connected to several distinct applications, or if some libraries are loaded accordingly to the device the user is running (very specific need I agree). I though it would be more "user-friendly" displayed like this than just inspecting the source code.

**Comment by deltakosh** [comment]
This is realllly specific :) This made me think you should have a library in a different repo for plugins like this one. Not part of the distribution but still available. What do you think?

**Comment by cubitouch** [comment]
Not a bad idea :)
I close this PR then.

**Comment by deltakosh** [comment]
And boooom :) 
https://github.com/MicrosoftDX/Vorlonjs/tree/master/plugins%20library
